certs/service.go

```diff
@@ -97,9 +100,12 @@ func (cs *certsService) RevokeCert(ctx context.Context, domainID, token, clientI
 		return revoke, errors.Wrap(ErrFailedCertRevocation, err)
```

Comment:
Why do we need to retrieve the client? If the client is non-existent will the error not be thrown by the certs repo?